packages/@glimmer/ssr/test/render-options-tests.ts

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+import { setComponentTemplate, createTemplate } from '@glimmer/core';
+import HTMLSerializer from '@simple-dom/serializer';
+import voidMap from '@simple-dom/void-map';
+import { SerializableNode } from '@simple-dom/interface';
+import { renderToString, RenderOptions } from '..';
+
+QUnit.module('@glimmer/ssr rendering', () => {
```

Comment:
I don't think these tests are running at all.

Reply:
It does if you run `testem ci`, which launches the Node process to run those on the command line. This setup was copied from `glimmer-lite-donut`, it could probably be a bit more explicit with a `yarn test:node` command, though I'm not sure how easy or hard it would be to get Testem to work that way.

It's also debatable if the SSR code should run in the browser, I could add the tests to the browser tests as well. I think it's probably useful, `renderToString` isn't really Node specific.